### DISCUSSION TITLE
Notes from the Laboratory: April 2023

### DISCUSSION THREAD

**Discussion by giscus** [davidsneighbour/kollitsch.dev#246] [Comments]
# Notes from the Laboratory: April 2023

This is the online playground of Patrick Kollitsch, David's Neighbour, in Germany produced, digital native living & working on Ko Samui, Thailand.

https://kollitsch.dev/blog/2023/notes-from-the-laboratory-april/

**Comment by brycewray** [comment]
> adding a search engine (I find myself often looking for that one entry where I wrote about that thing, or maybe I didn’t?)

If you haven’t yet tried [Pagefind](https://pagefind.app), it’s really hard to beat for single-site search.

**Reply by davidsneighbour** [reply]
I looked into Pagefind after Hugo Conf and just now again, and it grew nicely. I am not sold on it yet, though. What I like in, ahem, Algolia is features like typo tolerance. Also, [having to tag my metadata](https://pagefind.app/docs/metadata/) doesn't sit well with me. I wonder if it can be abused to create some form of individual non-public HTML index that Pagefind can use to create the index for the actual pages without all those data tags then. Hmm... `<brain status="on">`

**Reply by davidsneighbour** [reply]
Yeah... Ok... [I am convinced](https://kollitsch.dev/find/). Now the work starts.

**Reply by brycewray** [reply]
👍
